sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs

```diff
@@ -36,7 +36,7 @@ internal static class StorageVersionExtensions
         /// </summary>
         public const ServiceVersion LatestVersion =
 #if BlobSDK || QueueSDK || FileSDK || DataLakeSDK || ChangeFeedSDK
-            ServiceVersion.V2021_10_04;
+            ServiceVersion.V2021_12_02;
```

Comment:
I bumped `LatestVersion` for the Cold Tier private preview.  Undo this before this PR merges.

Reply:
Fixed.